app/views/support_interface/application_forms/show.html.erb

```diff
@@ -14,6 +13,11 @@
   </div>
 <% end %>
 
+<h1 class="govuk-heading-xl">
```

Comment:
Can you add this to `content_for :title` for consistency?

Reply:
You can't set `:title` and add a caption AFAIK?

Reply:
This should work:

```rb
<% content_for :title do %>
  <span class="govuk-caption-xl">Application #<%= @application_form.id %></span>
  <%= @application_form.candidate.email_address %> 
<% end %>
```

Reply:
Fixed in 39f29e7